skeleton/webpack/webpack.config.js

```diff
@@ -274,7 +273,7 @@ module.exports = ({ production } = {}, {extractCss, analyze, tests, hmr, port, h
       },
       {
         test: /\.scss$/,
-        use: ['css-loader', 'sass-loader'],
+        use: [...cssRules, ...sassRules],
         issuer: /\.html?$/i
```

Comment:
I have not used much less or stylus, but does it make sense to import these from TS/JS as well? If yes, then additional rules might be needed.

Reply:
Every file has two config, one for `issuer: /\.html?$/i` which is required css in html view. Another one is for `issuer: /\.(js|ts)?$/i` which is importing from ts/js. This one doesn't need style-loader because aurelia runtime does the work.

Reply:
scss/less/stylus works same in the view of webpack I think, their loader are chained to -> css-loader -> style-loader (for importing from ts/js).

Reply:
My bad, missed the `[tj]s` issuer.